parameters.cpp

```diff
@@ -437,13 +440,15 @@ bool P::addParameters() {
    RP::add("AMR.refine_on_restart","If true, re-refine vlasov grid on restart", false);
    RP::add("AMR.force_refinement","If true, refine/unrefine the vlasov grid to match the config on restart", false);
    RP::add("AMR.should_filter","If true, filter vlasov grid with boxcar filter on restart",false);
-   RP::add("AMR.refine_threshold","Determines the minimum value of the refinement parameter to refine cells", 1.0);
-   RP::add("AMR.unrefine_threshold","Determines the maximum value of the refinement parameter to unrefine cells", 0.0);
+   RP::add("AMR.use_alpha","Use alpha as a refinement index", false);
```

Comment:
Write out to some extent what "alpha" means, this is for the end user. So at least something about differences to neighbors.

Reply:
Still asking for more explanation here. :)

Reply:
Better?

Reply:
Yes. I am baffled that I don't see the discussion on the diff view even though it's not resolved yet. But well...